### ISSUE TITLE
Fix false positives for revert and revert-layer in font-family-no-missing-generic-family-keyword

### ISSUE THREAD

**Issue by mirisuzanne** [stylelint/stylelint#5850]
### What steps are needed to reproduce the bug?

We ran into this by setting:

```
.example {
  font-family: revert;
}
```

I see in the docs that `inherit` is allowed, but there are several other 'global keywords' that should also have exceptions:

- initial
- unset
- revert
- revert-layer

### What Stylelint configuration is needed to reproduce the bug?

```yaml
rules:
  font-family-no-missing-generic-family-keyword: true
```

### How did you run Stylelint?

CLI with `stylelint '**/*.scss'`

### Which version of Stylelint are you using?

`14.2.0`

### What did you expect to happen?

The global CSS keywords should not throw an error expecting a generic font-family.

### What actually happened?

A lint error was reported.

### Proposal to fix the bug

Exempt all global CSS keywords from needing a generic font-family

**Comment by ybiquitous** [comment]
@mirisuzanne Thanks for the report and using the template!

We can confirm the behavior on the [demo](https://stylelint.io/demo#N4Igxg9gJgpiBcIB0MAeBDAtgBwDYwAJgAdAOwIIDMJSAXAWkqwEtcBPeA50gCxgCdmtANxkK1OoxbtO3Ic3S5R5KjQZNMrDgQCupAM4wRY1ZI1bO-GADcBxlRPXTtV2-wa50bAcoC+ZEAAaEH02OnRUBHB9fSDwGkpmAHMokhViEH4dfH0MzjSKCgz0Biz8elIIej0Aa0qAd1I8glosmECTIpAAI1wIMBqKqpgcWjZm1p12zoIMyD7+Ifpua0VmKHo+VAm2jpUuyExMGElK+hHsMZ2pvcLZ8B19WghMemx+CGw7NiXNGO4kvRVotKHowLRmDRrtN9vdYGBPPwSpDSPRev1BmcoDo8MwwCUYPQwI9nq93p87MwYLkEC1djMMvDEciaGi+gMltjcfjaITyV93FSaZwANozCiTGF3CgFaVdZKVKzNMWwuVzGiGYkQ2z0Lm4PEE-T0epCHi65iUSgCE4MVa4KY08WFAC6Tv8sOdt0KjJgCPQSIhrPRHLO+h4EHcPHQpA2-O+9AgbkEsGFdJuDJAjikmnYnJx+p5hNIWGp0K9XSz5lzZz++gB9CSJwEeOzWnoNRgbHqEagZYzoNI4JRRMUYCWen02HQYBgGwpAYjfdhGQHQ9Z+tIMH9DaRUCppyqlQMtGjUH9Gz3VjXTVpkvL9w7bEoSOOut9zMDqLOzBwEZPdCXO4MmOPd0EYLdaB0KwKhLcdSDqCBGkA70QGLECd3Weh-S3I1v1IO11mQrosWpGcY3rSdfQtPEhHGW96WXVCqj1A1eSwhgf2wP96DKUt6PTRisXzVjCUMfBwQjVM7wzM4LjGeh9AgKCZyI+5ZNQVowMMTR5g1VSMjwgiLyU3pRM8MMiReY46CkhigKY5Z8LWWMIDrT92OWX93B47IYH0hzmH4Kwkmybd6h4IRqSnFT+KlFC43cH4zlqBob04aTGLE31nkWbBDB0KAqj9GI4IQpDYvvDIsok3L8sK858Gshhkvg1L-OqnL6DGL5Sra2kVTlDK5VlNUQAVCM-P6p17OJJ4XgakYbUdVUXTdGZPQzJ5BFIQEzk3eoN0m9K7JQvQhF6xC0rTOKukUbAoyBRQpiGE9P2VJ0MgFMiT0bDJ7xlab7jQGdLgABQ+AUIT40VAa6T5p1ojJAddFb3TuDbGJKHzyjkn5DrRGBqCVKaVuKXB6i8Gl-qIQGMmBmBLnelb7ODGociNdBKF5RZ9FgtDCVZ9m-th+5En4J4KmpXlexAZHqflJJFSOggBrlFDOe5yyjhtJHmYIFG5TRl1KpALHeJg198UMZoMj6eoBGFzHSl8i3ROiwkNYd2kyYptgjTrHbykOx37LNl2zlsGMIzeKxEm2CqMzD8ptLxCA+i-GADu4D2ua9zgfcpkOUNZok+gDwFuiRGdzlGPHs4JonlYMmwvZNkuEVc+tK+nItM-xz3+Bt03ycLoIM3bsuu6r3us83BuJqHxRfaNTBsghehg7HxiJ87na0WnhT3fn4n8+H5eFIBIPs6LroS4pUgp57yXZ5z7nF5Hv36FX3B183tv2UGPfR+1dJxPwHu-c+5cr6bhvvcO+XwH5727iAo+3RCYL29mfUeJtdIgjBO5Sor0URD0wNAAQN4cFp2jlsEc1tMF21bhmXBmw0AbxOEkWgPAh5hj-LA9U2tJC4w3vXNBjcmajSXtgp0I1pR01QCDWg4i1YVkChLTcTxZy62URjZRGRxonxViLKqYx8AbgYIcTAp5lraLWh6ShAiGDhUiqA6u3A6ywAgVIxis1SQx3nPJIR+NREYJhqTLBftYEAz1nIhRSjdGm1zosHx80EpXC3solR4tmpS00bLPWOi1Z6MViEwxetbqJK1k1SJsixoGHWKJS+hIgkAK0WrA20ojYUAKQcEk80QIKDeCUbmV1Vb2QAHoAAoRRgQAF7OmmfQGZAAGegABOZ0AAqAAlBM+gCzllrOdAAai2RsgAJNUmR9ljgxHQL9TBABRVAXxwSzgIMkzABB+noAIAARymPwNgBB+YtAgAQNBBAOzdHQN0Whk1bHoxwb0skEN4xTloMMuJKFJn7PmbMlZ6ztm7P2QS45pyLnpOlFclCNzeb3NPk8l50t3nIoIKkoFILnjgsIFCmFcLWl3E6frE2TJ-QslRNCveLjX55zUi3QelKuiioDMOSVgJpXHybuEqmGZlXirZBiBSIxU7p2fv3RJnjP7f1-tfRVcJ3xivciXFOulUQavAZgyRn8oFNNtSKh1KqgwAKNTpKhbrUHoIMc3NwfCQB6qdcGn1wi56WNQG+P0gaDDNAAIz+ozfqkumlWAUWNa6y1OrGLxtVcGs4VhsQxmjB4GgSQowxj8ZDIU-kq2sldWa+uHrT5epXmvZgybJp5o-MOXt7qLWeo-v7RpY7Y3dtRNOiNYjMGbhjXan0+b3KBJEZG5Woz1bzsubTEA9NGYkwyfcAe6bJ1QjtaNMWaickyzlhe-Rx6jEJM1hYnWz6aluPqRfQOvq56swFdKdpgr1om0rM4V29B-kQF5KmAun9wrWh4r6KyJxYAywQ-gqdVkwL7XNW-Od59rWjr-v2EjPayOHzAbOwd86wNJGgeOhjg53IWK0uukp0bGGMVXO5VNNdLh13Uc0JZxG+PDn5vy+hiFRP2XE8OKcVg6B8EMLhPu9cQMeOo5TL+I6l07szIx1E2mbR6epCx1xdSTOny3QITj3HY2adZFBXAKGdBoehnes9VmfOoiceho+A6QvL1jTwZ6sAkhWCLGh8Vi9wOxu4LAcIb1aQACYTaPmfCWXCsF0WYpvTUnFsy8WLNJUSvZ+LDknPOee6JIBaV3K1Yy7KbzisvkIJysFELeWwqtvC1G8GMz4B2pwxZAhDxVDOoohOjFJMHtk7SXNGZJP41mxwrh238vyd27OAZVoShQUJGu1jVH2Nxas988CV3oK3ersEqNqF5Wxue5dyC0FlMTaHgwhVJs-sQWu8hiOBHo7vEJsweOx0BLXPO2Bf7UO7O6epI5jVxmtXubB2d0CL2AeEiRDtQk84SjRxnfd2LXjUck4x9BCnjYEwChpzzITX2h2-bRwFgQdcJYCb7XPGLmHh0-zo364nAz-lC+ESL5jdPZWS88xBnj62BcK8BUr8xKuecE5+1Z2StdGCqI8La5HN01LDDTsaCKUWe7+RrMwf4e8CMJkoApJS-Bq4Uet9dE2pAdCYDQYsfA6A9x7xmYt8tsbQ-h485J+HYB3fENpAAFhD2HiPSwi0bkBHHj4tkUfxVRYlFTp9QexvZUsSOhVcqx0R-5c2m2BZHqxbdDjtHLPU2pV0K9q3Qm3pXJbyWGiP35Plvcb93ean3oAwBPJNipt2M2jAcSnUhmCG6DoNiyCOy0H9kffHQ9Cexo6tHXfzB99sWp51VXCq3Mm5NtfxYt-79U850-o3F+39N9t8b8MU98D9CRUN0ME8rMP9S50ASoKtyEF8MgatFk6sDlCUdkmt6sWtyV2t4kut6V7hetXkoB3lzJ9ACAP9QVuVIUYBoVxt4DJtDZptMot9spo5DhuhuAucnMZUX8GcIkYD2CaotZuDiw-8n5PstU+dhDgDFgUwyIzxJAuCeDOo9pWRpV2oRDOpMJED+ARkPoQBUC5kSVDlGszD1lWsKUB8L1CCetnk+syD1gqCdCIwaDRt6C+VgdV8OlWD7JYD9RlcjhyNDNxc2NBCK0Ai3DI93cDcQi+DNUADt138Yiv4IgpN5JDpUxTs2D5CG9Ydm8Eckdg8gCODaoYACoioKDq97ha85Dyi3g6pqj4CjQsdOEcdT8n5z9N1AC8jGi8pKj6p2CmpajbY1Mid+jRDBiqiFpRje1CF0tMFCodBTIr80juobsmCQcJir9wxIxTx214xa1Zw9BlDbQnpgshoUItp6xIDgtGQTJ8BY0VsxiQB6iTY7RnpHxux+ANgfCa9dirMvjfVgjLExd+Dy1zNpd+8MwQT9dKlBM7s1dtUNdYTGJ4Sgj4jwSNVpDkj1MUJMS4j0i00O8cjPjLiCio4ii45oQTB3RfAQBfAgA); only `revert` and `revert-layter` emits the error though.

Maybe we need to update the following keyword set:

https://github.com/stylelint/stylelint/blob/915ffffb785daea9e02f24d346ad8d2f52c8eb18/lib/reference/keywordSets.js#L68

I've labeled the issue as ready to implement. Please consider [contributing](https://stylelint.io/contributing) if you have time.

There are [steps on how to fix a bug in a rule](https://stylelint.io/developer-guide/rules#fix-a-bug-in-a-rule) in the Developer guide.

**Comment by ybiquitous** [comment]
Ref: https://www.w3.org/TR/css-cascade-5/#revert-layer

**Comment by mirisuzanne** [comment]
Thank you!

**Comment by jeddy3** [comment]
Released in [14.3.0](https://stylelint.io/changelog/#1430).
